package-lock.json

```diff
@@ -8549,7 +8549,7 @@
         "chromedriver": "bin/chromedriver"
       },
       "engines": {
-        "node": ">=16"
+        "node": ">=18"
```

Comment:
Looks like the chromedriver update to 118 has some dependencies on Node Version , Not sure if this will be issue but the Test are successful

Reply:
It does fire a warning when it runs - the node 18 upgrade is on the roadmap.